setup.py

```diff
@@ -40,6 +40,7 @@
             'scrapbook~=0.5.0',
             'jupyter',
             'graphviz',
+            'pytest-socket>=0.2',
```

Comment:
Is this lower bound tested, or just a guess?

Reply:
The fixture used in this PR (`socket_disabled`) was [only added to the API in 0.2](https://github.com/miketheman/pytest-socket/compare/0.1.0...0.2.0). I added a comment to clarify this.